lib/src/view/broadcast/broadcast_game_screen.dart

```diff
@@ -354,6 +355,7 @@ class _BroadcastBoardState extends ConsumerState<_BroadcastBoard> {
           newShapeColor: boardPrefs.shapeColor.color,
         ),
       ),
+      boardPrefs: boardPrefs,
```

Comment:
nit: maybe we add this new line right under the size argument to keep argument order the same